frontend/components/groups/js/groups.component.ts

```diff
@@ -54,6 +54,7 @@ class GroupsController implements ng.IController {
 	private savedGroupData: any;
 	private customIcons: any;
 	private lastColorOverride: any;
+	private lastColorOverrideAll: any;
```

Comment:
I don't think this is the best approach.

There's 2 problems, 
- another class scope variable which is not desirable
- when you call colorOverrideAllGroups(), it runs the computation again for all the groups (even though we know for a fact that n-1 groups are already overridden

I think it's better if you have a flag passed into `GroupService.removeColorOverride()`, telling it to ignore resetting overrideAll, which wouuld only set to true if we are removing override because we entered the edit page

Let me know what you think! 😄